lib/vdsm/storage/lvm.py

```diff
@@ -1910,6 +1910,10 @@ def changeLVsTags(vg, lvs, delTags=(), addTags=()):
         raise se.LogicalVolumeReplaceTagError.from_lvmerror(e)
 
 
+def reload_lvs(vg_name):
+    _lvminfo._reloadlvs(vg_name)
```

Comment:
This is not needed since we have lvm.invalidateVG(). It is already used in many places to make sure we load fresh data.

This is also not consistent with the way lvm module works. Maybe reloading instead of invalidating the cache is better (I don't see why), but adding inconsistent behavior does not sound like the right way.